apps/wallet-dashboard/components/Dialogs/Staking/timelock/UnstakeTimelockedObjectsDialog.tsx

```diff
@@ -0,0 +1,252 @@
+                            handleCopySuccess={handleCopySuccess}
+                        />
+                    </div>
+                </LayoutBody>
+                <LayoutFooter>
+                    <Button
+                        onClick={handleTimelockedUnstake}
+                        text="Unstake"
+                        icon={
+                            isTransactionPending || isUnstakeTxLoading ? (
+                                <LoadingIndicator />
+                            ) : undefined
+                        }
+                        disabled={
+                            !timelockedUnstakeTx || isTransactionPending || isUnstakeTxLoading
+                        }
+                        type={ButtonType.Secondary}
+                        fullWidth
+                    />
+                </LayoutFooter>
+            </Layout>
+        </Dialog>
+    );
+}
+
+interface TimelockStakeObjectsOverviewProps {
+    groupedTimelockedObjects: TimelockedStakedObjectsGrouped;
+    handleCopySuccess: () => void;
+    rewardsPool: string | undefined;
+}
+
+function TimelockStakeObjectsOverview({
+    groupedTimelockedObjects: { stakes, validatorAddress, stakeRequestEpoch },
+    handleCopySuccess,
+    rewardsPool,
+}: TimelockStakeObjectsOverviewProps) {
+    const stakeId = stakes[0].timelockedStakedIotaId;
+
+    return (
+        <>
+            {stakeId && (
+                <ValidatorStakingData
+                    key={stakeId}
+                    validatorAddress={validatorAddress}
+                    stakeId={stakeId}
+                    isUnstake
+                />
+            )}
+
+            <Panel hasBorder>
+                <div className="flex flex-col gap-y-sm p-md">
+                    <KeyValueInfo
+                        keyText="Stake Request Epoch"
+                        value={stakeRequestEpoch}
+                        fullwidth
+                    />
+                    {rewardsPool && (
+                        <KeyValueInfo keyText="Rewards" value={rewardsPool} fullwidth />
+                    )}
+                    <KeyValueInfo keyText="Total Stakes" value={stakes.length} fullwidth />
+                </div>
+            </Panel>
+
+            {stakes.map((stake, index) => (
+                <Collapsible
+                    initialClose
+                    key={stake.timelockedStakedIotaId}
+                    title={'Stake' + ' ' + (index + 1)}
+                >
+                    <Panel>
+                        <div className="flex flex-col gap-y-sm p-md--rs py-sm">
+                            <KeyValueInfo
+                                keyText="Stake ID"
+                                value={formatAddress(stake.timelockedStakedIotaId)}
+                                hoverTitle={stake.timelockedStakedIotaId}
+                                onCopySuccess={handleCopySuccess}
+                                copyText={stake.timelockedStakedIotaId}
```

Comment:
I don't get the notification when I copy it

Reply:
z index issue. Won't change it here cause it's being fixed in another PR